core/src/main/java/com/sequenceiq/cloudbreak/reactor/handler/cluster/StopStartUpscaleCommissionViaCMHandler.java

```diff
@@ -96,7 +96,7 @@ protected Selectable doAccept(HandlerEvent<StopStartUpscaleCommissionViaCMReques
                     String.valueOf(allInstancesToCommission.size()));
 
             ClusterSetupService clusterSetupService = clusterApiConnectors.getConnector(stack).clusterSetupService();
-            clusterSetupService.waitForHostsHealthy(new HashSet<>(allInstancesToCommission));
+            allInstancesToCommission = clusterSetupService.waitForHostsHealthy(new HashSet<>(allInstancesToCommission));
```

Comment:
we should have new variable called healthyInstancesToCommision so that it's clear that we are only going to commission the healthy nodes